storagetransfer/api/StorageTransfer.Samples/QuickstartSample.cs

```diff
@@ -36,8 +35,8 @@ public TransferJob Quickstart(
                 ProjectId = projectId,
                 TransferSpec = new TransferSpec
                 {
-                    GcsDataSink = new GcsData { BucketName = sourceBucket },
-                    GcsDataSource = new GcsData { BucketName = sinkBucket }
+                    GcsDataSink = new GcsData { BucketName = sinkBucket },
+                    GcsDataSource = new GcsData { BucketName = sourceBucket }
                 },
                 Status = TransferJob.Types.Status.Enabled
             };

```

Comment:
On line 46 below, `RunTransferJob` is a long running operation but it's not being polled or otherwise tested for status.

Reply:
That's actually expected consistent with other languages, we just check that the transfer job got created instead of blocking to wait for any specific status. The CheckLatestTransferOperation sample shows how to poll

Reply:
OK, thanks